### PR TITLE
Add --write-buffer-size and --check-download-hash to common parser

### PR DESCRIPTION
**Pull request by agoncharov-reef**

### CODE REVIEW

File: b2/console_tool.py [hunk 1]
```diff
@@ -498,6 +498,10 @@ def get_parser(cls, subparsers=None, parents=None, for_docs=False):
                 common_parser.add_argument('--verbose', action='store_true', help=argparse.SUPPRESS)
                 common_parser.add_argument('--logConfig', help=argparse.SUPPRESS)
                 common_parser.add_argument('--profile', default=None)
+                common_parser.add_argument('--write-buffer-size', type=int, help=argparse.SUPPRESS)
+                common_parser.add_argument(
+                    '--check-download-hash', action='store_true', help=argparse.SUPPRESS
```
Comment (ppolewicz) [inline b2/console_tool.py:503]: checking hash is enabled by default, the option should be reversed, like `--skip-hash-verification`

File: b2/console_tool.py [hunk 2]
```diff
@@ -498,6 +498,10 @@ def get_parser(cls, subparsers=None, parents=None, for_docs=False):
                 common_parser.add_argument('--verbose', action='store_true', help=argparse.SUPPRESS)
                 common_parser.add_argument('--logConfig', help=argparse.SUPPRESS)
                 common_parser.add_argument('--profile', default=None)
+                common_parser.add_argument('--write-buffer-size', type=int, help=argparse.SUPPRESS)
+                common_parser.add_argument(
```
Comment (ppolewicz) [inline b2/console_tool.py:502]: these should be documented on `download-file-by-*` and `sync`, perhaps just add them there and not to global parser

File: b2/console_tool.py (hunk 1 above)
Comment (agoncharov-reef) [reply b2/console_tool.py]: Good catch!

**Comment by ppolewicz** [comment]
This is ready to go, please submit to upstream

**Pull request closed without merging**